### PR TITLE
update Falco charts for upcoming Falco 0.38

### PR DESCRIPTION
**Pull request by alacuku**
**What type of PR is this?**

> Uncomment one (or more) `/kind <>` lines:

> /kind bug

> /kind cleanup

> /kind design

> /kind documentation

> /kind failing-test

> /kind feature

> If this PR will release a new chart version please make sure to also uncomment the following line:

/kind chart-release

**Any specific area of the project related to this PR?**

> Uncomment one (or more) `/area <>` lines:

/area falco-chart

> /area falco-exporter-chart

> /area falcosidekick-chart

> /area event-generator-chart

> /area k8s-metacollector

**What this PR does / why we need it**:

This PR updates the Falco charts for the upcoming Falco 0.38.0
**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Checklist**

- [ ] Chart Version bumped
- [ ] Variables are documented in the README.md
- [ ] CHANGELOG.md updated

### CODE REVIEW

File: charts/falco/Chart.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 apiVersion: v2
 name: falco
-version: 4.3.1
-appVersion: "0.37.1"
+version: 4.4.0
+appVersion: "0.38-rc3"
```
Comment (LucaGuerra) [inline charts/falco/Chart.yaml:4]:
```suggestion
appVersion: "0.38.0-rc3"
```

Will be replaced with `"0.38.0"` in the end though

Comment (leogr) [reply charts/falco/Chart.yaml:4]: I'll apply `0.38.0`

Comment (leogr) [inline charts/falco/Chart.yaml:4]:
```suggestion
appVersion: "0.38.0"
```

File: charts/falco/Chart.yaml
Comment (FedeDP) [inline charts/falco/Chart.yaml:1]: I'd also bump `falcoctl` image to v0.8.0 while we are at it

File: charts/falco/values.yaml
Comment (FedeDP) [inline charts/falco/values.yaml:1]: I'd also update falcoctl image to 0.8.0 while we are at it: https://github.com/falcosecurity/charts/blob/ba62cde02297bbf70578bc84316587deeccf89fa/charts/falco/values.yaml#L379

Comment (leogr) [reply charts/falco/values.yaml:1]: good catch. Fixed. Tnx.

**Review by leogr — APPROVED** [review]
/hold cancel

**Pull request merged**